src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -1023,7 +1023,7 @@ private static BlockEntry<ActiveBlock> createFireboxCasing(BoilerFireboxType typ
             .tag(BlockTags.LOGS_THAT_BURN, BlockTags.OVERWORLD_NATURAL_LOGS)
```

Comment:
where's the rubber logs block tag?

Reply:
The only places where the block tag gets used is in the `LOGS_THAT_BURN` block tag, which rubber logs already have, and when it gets copied to the respective item tag, so I decided the block tag wasn't needed.